dev-infrastructure/configurations/svc-cluster.bicepparam

```diff
@@ -11,6 +11,8 @@ param disableLocalAuth = false
 param deployFrontendCosmos = false
 param deployMaestroInfra = false
 param maestroNamespace = 'maestro'
+param deployCsInfra = true
```

Comment:
```suggestion
param deployCsInfra = false
```
I think we want this to default to false by convention

Reply:
good point. fixed